src/actions/index.js

```diff
@@ -6,3 +6,17 @@
 // our action creator will be a function that returns a function
 // the url to fetch characters from is `https://swapi.co/api/people/`
 // remember that now we have controll over our thunk-based action creator
+import axios from 'axios';
+
+export const FETCH_START = 'FETCH_START';
+export const FETCH_SUCCESS = 'FETCH_SUCCESS';
+export const FETCH_FAILURE = 'FETCH_FAILURE';
```

Comment:
Instead of making your action type names general I'd provide more context in them.

_e.g_ 
instead of `export const FETCH_START = 'FETCH_START';`
I would do something like export const FETCH_STARWARS_CHAR_DATA= "FETCH_STARWARS_CHAR_DATA";

It may be more verbose but it's more descriptive which can help a lot in bigger applications.